migrations/staging/20171009120412_stg_add_oral_reports_to_court_reports.js

```diff
@@ -1,15 +0,0 @@
-
```

Comment:
Can we keep this file and just add additional fields in a new one. We should be following a "Fix forward" migration process now that we're running migrations on all environments.

Reply:
Ah, I thought this had only made it as far as the merge branch, not master. Will update now